clang/test/CIR/Lowering/ThroughMLIR/declaration.c

```diff
@@ -0,0 +1,29 @@
+// RUN: split-file %s %t
+// RUN: %clang_cc1 -fclangir -fno-clangir-direct-lowering -emit-mlir=core %t%{fs-sep}test_declaration.c 
+// RUN: FileCheck --input-file=%t%{fs-sep}test_declaration.mlir %t%{fs-sep}test_declaration.c
```

Comment:
You can merge both files into one, and test all of them in one go, no need to split files.